src/AspNet.Security.OAuth.LinkedIn/LinkedInAuthenticationExtensions.cs

```diff
@@ -0,0 +1,53 @@
+﻿ // ReSharper disable once CheckNamespace
+
+using System;
+using AspNet.Security.OAuth.LinkedIn;
+using Microsoft.Framework.DependencyInjection;
+using Microsoft.Framework.Internal;
+using Microsoft.Framework.OptionsModel;
+
+namespace Microsoft.AspNet.Builder {
+    public static class LinkedInAuthenticationExtensions {
+        public static IServiceCollection ConfigureLinkedInAuthentication(
+            [NotNull] this IServiceCollection services,
+            [NotNull] Action<LinkedInAuthenticationOptions> configuration)
+        {
```

Comment:
I'm not sure why braces style is so inconsistent. I probably missed something when generating the `.DotSettings` file.